src/cockpit/askpass.py

```diff
@@ -66,7 +66,13 @@ def main() -> None:
         sys.exit('This command must be run with stdin connected to a socket.')
 
     interaction = AuthorizeInteraction(args.prompt)
-    asyncio.run(interaction.run(connection))
+    loop = asyncio.get_event_loop()
+
+    # shim for Python < 3.7
+    if not hasattr(asyncio, 'get_running_loop'):
+        asyncio.get_running_loop = lambda: loop
```

Comment:
This still annoys me. I'll see to moving that to polyfills as well.

Reply:
I tried that.. but it introduces a conflict with systemd_ctypes, is racy due to the global instantiation of an event loop, and does extra work. So I think it's less evil as it is here.